setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 
 project = "cytominer_database"
-version = "0.3.3"
+version = "0.3.5"
```

Comment:
any reason to not use 0.3.4?

Reply:
Thanks for double checking on this - I think the version patch for `0.3.4` might have been missed here during an earlier point in the source (the current version is `0.3.4`, [see here](https://github.com/cytomining/cytominer-database/releases) and in the [`CITATION.cff`](https://github.com/cytomining/cytominer-database/blob/master/CITATION.cff#L29), so we need to bump to `0.3.5`).